tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
 commands =
     shed
     flake8
-    mypy --config-file=tox.ini .
+    ; mypy --config-file=tox.ini src/hypothesmith/
```

Comment:
It works now! Thanks!

Looks like you forgot to uncomment this line.

Reply:
I actually just decided that it wasn't worth hours to get the current version of Mypy passing when the project is in minimal maintenance mode anyway 😅

Maybe if I had a weekend free, but I wanted to get this fix out and then update hypothesis-jsonschema instead - which is _unavoidably_ a bigger pain.